tests/data/ecoli_small_experiments.toml

```diff
@@ -88,7 +90,7 @@ formation_energies = [
   {metabolite_id = "fdp", location = -2220.9, scale = 2.1},
   {metabolite_id = "g3p", location = -1106.4, scale = 1.3},
   {metabolite_id = "dhap", location = -1111.9, scale = 1.1},
-  {metabolite_id = "pep", location = 0, scale = 1},
+  {metabolite_id = "pep", location = -1207.5, scale = 1},
```

Comment:
I think I changed it so that the formation energies would generally get closer together and the rescaling would be a little less extreme. Can't remember if it made a big difference but seems like we might as well keep it